test/main/modules.js

```diff
@@ -81,14 +81,10 @@ QUnit.module( "after (skip)", {
 	}
 } );
 
-QUnit.test( "does not run after initial tests", function( assert ) {
```

Comment:
I'm not sure that this is correct. Looking at it locally, this means the `after` hook never runs, which would be a regression. During the first/last actually running test, it previously checked if we're currently executing the last non-ignored test. Which meant that if the last registered test is skipped, the after hook will run correctly after the last actually run test.

However, that logic was removed as part of this patch, and thus we now get a similar bug but affecting the `after` hook rather than the `moduleDone` callback.

I think for this to work we need to continue to "look ahead" like it did before, so that it knows during this test that it will be the last.

Reply:
Agreed, good catch @Krinkle! Let me see what I can do

Reply:
@step2yeung Friendly ping to check if you're able to find time for this. If not, I might give it a go next week for the next patch release. Thanks again!

Reply:
Thanks for the reminder Krinkle! Will get back to it this week!

Reply:
One more reminder 🙂. No problem at all, but I'll give this a look this Saturday/Sunday if I don't hear back. Thanks again.